newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -40,12 +43,13 @@ export default function SummaryPage() {
   const userInfo = useSelector(getUserInfo);
   const isCreator = userInfo !== null && newdle !== null && userInfo.uid === newdle.creator_uid;
   const dispatch = useDispatch();
+  const hasParticipants = useSelector(getNumberOfParticipants);
   const [_sendResultEmails, mailSending, mailError, sendMailResponse] = client.useBackendLazy(
     client.sendResultEmails
   );
   const [_setFinalDate, submitting] = client.useBackendLazy(client.setFinalDate);
   usePageTitle(newdle && `Summary: ${newdle.title}`, true);
-
+  const [checked, setChecked] = useState(false);
```

Comment:
I don't think this is a good name - if it was in a component which renders a checkbox but nothing else then it'd be fine, but in a large component like this one the name should be more meaningful.

Maybe `gridViewActive` or something like this?